src/v3/src/transformer/layout/securityQuestion/securityQuestionEnroll.ts

```diff
@@ -46,14 +46,15 @@ export const transformSecurityQuestionEnroll: IdxStepTransformer = ({ transactio
   const { uischema, data, dataSchema } = formBag;
 
   const predefinedQuestionOptions = inputs?.[0]?.options?.filter(({ value }) => !(value as Input[])?.some(({ name }) => name === 'question'));
+  const customQuestionOptions = inputs?.[0]?.options?.filter(({ value }) => (value as Input[])?.some(({ name }) => name === 'question'));
```

Comment:
We were re-using the predefined questions elements for custom answer field. This causes a problem when errors are set because when an error is set on the custom element, it will not surface to the UI since the predefined answer element is used. (This fixes this issue).